### PR TITLE
add assignment pipeline value

### PR DESCRIPTION
**Pull request by johannbarbie**

### CODE REVIEW

File: PIPS/pip-008.md [hunk 1]
```diff
@@ -12,11 +12,15 @@ This PIP is basically a set of amendments to PIP004, PIP005, PIP007.
 ## Motivation
 When we are running out of cash, our top priority should be to look for more money to sustain the project. Building positive cashflow, looking for new investments and/or grants. We cut our expenses to give us more time for that and to postpone the bankruptcy.
 
-## Applicability
-This PIP applies the next Monday 00:00 UTC after Escrow council balance becomes less than 50,000 DAI worth of ETH.
-This PIP cease to apply the next Monday 00:00 UTC after Escrow council balance has more than 80,000 DAI worth of ETH.
-
 ## Specification
+
+**Assignment Pipeline Value** - the sum of revenue expected from all ongoing project with low execution risk.
```
Comment (troggy) [inline PIPS/pip-008.md]: should we address reaching bankrupcy long before the actual payout?

Comment (troggy) [reply PIPS/pip-008.md]: e.g. if we expect payout next year

Comment (johannbarbie) [reply PIPS/pip-008.md]: maybe find a better wording for `low execution risk`

**Comment by troggy** [comment]
Overall, I don't mind except "Assignment Pipeline Value" is super loosely defined

File: PIPS/pip-008.md (hunk 1 above)
Comment (mateleshkavo) [reply PIPS/pip-008.md]: We need to add an additional responsibility to the treasurer role to calculate this number and write down it here. E.g. - treasurer holds responsibility of calculating this number, also he needs to add our level of confidence in this number.

**Comment by johannbarbie** [comment]
screen shot 2018-04-23 at 6 36 49 pm

**Pull request merged**